packages/web-components/src/components/cbp-nav-item/cbp-nav-item.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, Element, Host, Prop, h } from '@stencil/core';
+import { setCSSProps } from '../../utils/utils';
+@Component({
+  tag: 'cbp-nav-item',
+  styleUrl: 'cbp-nav-item.scss',
+})
+
+export class CbpNavItem {
+
+  @Element() host: HTMLElement;
+
```

Comment:
You should have a prop to denote the active/current state. Maybe "selected" to match others? This is reflected and used as a style hook for the "current" nav item. You also need to apply aria-current to the controls via the DOM (since it's slotted, not rendered) and manage removing aria-current when one is clicked and made the new current item. You can reference tabs for this.

Reply:
updated as discussed using the tab(s) as an example